packages/assets/scss/03-organisms/_rich-text.scss

```diff
@@ -60,11 +60,11 @@ $heading-indent: $gutter;
   & &__footnote {
     border-style: solid;
     border-width: 2px;
-    font-size: .6em;
+    font-size: .7em;
```

Comment:
```suggestion
    font-size: $fonts-xsmall;
```

For font sizes, we standardized on using rem instead of em. 1rem is slightly larger than 0.7em, but I think we need that extra size because it's slightly under the required size on mobile

Reply:
![Screen Shot 2024-02-20 at 10 24 08 AM](https://github.com/massgov/mayflower/assets/5789411/846f0484-627a-4558-a29d-ce6db3c300e8)
slightly under the size on mobile